### PR TITLE
Merge nodejs/master

### PR DESCRIPTION
**Pull request by kunalspathak**
##### Checklist

- [x] `make -j4 test` (UNIX), or `vcbuild test` (Windows) passes
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)

Merge 5bda5fa as of 03-17-2017

Commit to review:
* 2a2d9d1 : Removed TTD code after refactoring of `node_file.cc`.

### CODE REVIEW

**Comment by kunalspathak** [comment]
@digitalinfinity - can you review?

CI  : https://ci.nodejs.org/job/chakracore-test-windows/43/

File: src/node_file.cc [hunk 1]
```diff
@@ -564,11 +564,6 @@ static void Stat(const FunctionCallbackInfo<Value>& args) {
   ASSERT_PATH(path)
 
   if (args[1]->IsObject()) {
-
-#if ENABLE_TTD_NODE
```
Comment (digitalinfinity) [inline src/node_file.cc]: These are removed? Is that right? cc @mrkmarron

**Review by digitalinfinity — CHANGES_REQUESTED** [review]
LGTM other than the TTD changes...

File: src/node_file.cc (hunk 1 above)
Comment (kunalspathak) [reply src/node_file.cc]: Yes, Already confirmed from @mrkmarron before removing this.

Comment (mrkmarron) [reply src/node_file.cc]: They are no longer explicitly modifying the raw double* so everything is tracked by the JsRT TTD hooks now.

**Pull request merged**